python/requirements_ml_docker.txt

```
@@ -15,3 +15,7 @@ torch-sparse==0.6.12
 # torch-geometric has to be installed after torch-scatter and torch-sparse.
 torch-geometric==2.0.3; python_version < '3.7'
 torch-geometric==2.0.4; python_version >= '3.7'
+
+# Jax
+-f https://storage.googleapis.com/jax-releases/jax_releases.html
+jax[cuda]
```

```suggestion
jax[cuda]==0.1.69
```

maybe also with a 'please sync versions' note like on L6?

---

this poses another challenge: the `[cuda]` extra was only [added](https://github.com/google/jax/commit/4d8bce1b8515c68a8432d379d964ea3e6bea463c) in `0.2.23` ([blame](https://github.com/google/jax/blame/a91cb8161340650c2f45f455ab9b5a1178de7658/setup.py#L66)). the `0.1.69` dist does not contain any extras.

py36 support was [dropped](https://github.com/google/jax/commit/94446ff757a3ff1c0dfaba3ecff437ae343c2c71) in `0.2.18`, so another possibility to keep cpu and gpu reqs in sync would be to use [e.g.](https://github.com/google/jax/blob/jax-v0.2.17/setup.py#L19) `jax[cuda111]==0.2.17` here and `jax[cpu]==0.2.17` in `python/requirements/ml/requirements_dl.txt`. that way, you don't need to specify `jaxlib` at all anymore

---

thanks @ddelange, will take a look soon!